argilla-server/pyproject.toml

```
@@ -22,7 +22,8 @@ maintainers = [{ name = "argilla", email = "contact@argilla.io" }]
 dependencies = [
     # Basic dependencies
     "fastapi ~= 0.115.0",
-    "pydantic ~= 1.10.18",
+    "pydantic ~= 2.9.0",
```

Can we upgrade to the latest version?

```suggestion
    "pydantic ~= 2.9.2",
```

Don't forget to do `pdm install` after change this, so the lock file is updated.

---

We don't need to point to the latest patch release. The `~=` operator will match the latest compatible version. See [here](https://peps.python.org/pep-0440/#compatible-release) for more info